### PR TITLE
Magiclysm: Forest Tomb

### PR DESCRIPTION
**Pull request by arijust**
#### Summary

```SUMMARY: Mods "Magiclysm: Forest Tomb"```
#### Purpose of change

Add new low-tier dungeon to Magiclysm.
#### Describe the solution

Add forest tomb, small tomb in forest with few enemies, but decent chance to get animist books and low-tier magic tools,
#### Describe alternatives you've considered

Stay content with doing monsters and furniture only, if this location is poor quality too.

#### Additional context

I made pull request earlier to see reaction for that one.
I also need to add some monsters and add loot before actually releasing this.

### CODE REVIEW

**Comment by mkikt4743** [comment]
seems cool and good

File: data/mods/Magiclysm/worldgen/forest_tomb.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        "#######.....############",
+        "###########..###########",
+        "###########..###########",
+        "############.###########",
+        "###########..###########",
+        "###########..###########",
+        "###########.############",
+        "########........########",
+        "########........########",
+        "########...HZ...########",
```
Comment (Maleclypse) [inline data/mods/Magiclysm/worldgen/forest_tomb.json]: Should H and Z have a definition in the palette? I couldn't find them. I'm excited to see this.  Forests feel so empty sometimes.

File: data/mods/Magiclysm/worldgen/forest_tomb.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        "                  S     ",
+        "            S           ",
+        "   S                    ",
+        "                    S   ",
+        "        S               ",
+        "   S            S       ",
+        "   ##################   ",
+        "   ##################   ",
+        " S ####..##>>##...###   ",
+        "   ###$..........####S  ",
```
Comment (Maleclypse) [inline data/mods/Magiclysm/worldgen/forest_tomb.json]: Also can't find x and $ in definitions.  Are they monsters?

File: data/mods/Magiclysm/worldgen/forest_tomb.json [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        "   ###...##..#...x### S ",
+        "  S####x###..########   ",
+        "   ########..########   ",
+        "          T  T     S    ",
+        "     S                  ",
+        "          T  T       S  ",
+        "      S                 ",
+        "          T  T          ",
+        "                 S      ",
+        "    S                   "
```
Comment (Maleclypse) [inline data/mods/Magiclysm/worldgen/forest_tomb.json]: Looks like Travis feels you need one more row of terrain?  I'm not sure why but that's what the test looked like to me.

File: data/mods/Magiclysm/worldgen/forest_tomb.json (hunk 1 above)
Comment (arijust) [reply data/mods/Magiclysm/worldgen/forest_tomb.json]: H is going to be altar.
Z is for boss/main monster.

File: data/mods/Magiclysm/worldgen/forest_tomb.json (hunk 2 above)
Comment (arijust) [reply data/mods/Magiclysm/worldgen/forest_tomb.json]: $ is weaker monsters.
x is loot.
I will probably reverse these two.

File: data/mods/Magiclysm/worldgen/forest_tomb.json [hunk 4]
```diff
@@ -0,0 +1,138 @@
+[
+  {
+    "type": "palette",
+    "id": "tomb",
+    "terrain": {
+      ".": [ "t_rock_floor" ],
```
Comment (curstwist) [inline data/mods/Magiclysm/worldgen/forest_tomb.json]: can remove the [ ] for all of these except ones that have an array of terrains. Format will be:
`".": "t_rock_floor",`

**Comment by arijust** [comment]
Closing this for certain amount of time.
I need to redo this location at new git branch.

**Comment by KorGgenT** [comment]
it seemed like it was coming together, so i hope to see it soon

**Pull request closed without merging**